cl_sii/rtc/data_models_aec.py

```diff
@@ -648,6 +653,49 @@ def slug(self) -> str:
     # Custom Methods
     ###########################################################################
 
+    def verify_signature(
+        self,
+        xml_doc: xml_utils.XmlElement,
+    ) -> Optional[bool]:
+        """
+        Verify signature of AEC XML document ``xml_doc``.
+        :param xml_doc: An XML document, as returned by ``xml_utils.parse_untrusted_xml()``.
```

Comment:
We should add an explanation that `:param xml_doc:` isn't just any XML document; it should be the AEC XML document from which `AecXml` was parsed.

Reply:
> We should add an explanation that `:param xml_doc:` isn't just any XML document; it should be the AEC XML document from which `AecXml` was parsed.

I've been thinking about this, and I think that the close relationship between `xml_doc` and the `AecXml` instance would be clearer if we moved this method out of the `AecXml` class.

My suggestion is to move method `AecXml.verify_signature` to `cl_sii.rtc.xml_utils` and change the type signature to `verify_signature(aec_xml_doc: cl_sii.libs.xml_utils.XmlElement, aec_xml: cl_sii.rtc.data_models_aec.AecXml) -> Optional[bool]`.

Reply:
I wasn't really quite comfortable with adding the method in `AecXml`, so happy to move it to `cl_sii.rtc.xml_utils`